backend/curriculum_tracking/models.py

```diff
@@ -1383,3 +1385,20 @@ def add_collaborator(self, user, add_as_project_reviewer):
                 if self.recruit_project:
                     self.recruit_project.reviewer_users.add(user)
         self.save()
+
+    def get_users_that_reviewed_since_last_review_request(self):
+        if self.review_request_time is None:
+            return []
+
+        if self.content_item.content_type == ContentItem.PROJECT:
+            ReviewClass = RecruitProjectReview
+
+        elif self.content_item.content_type == ContentItem.TOPIC:
+            ReviewClass = TopicReview
+
+        reviews = ReviewClass.objects.filter(
+            timestamp__gte=self.review_request_time
+        )
+
+        return list(map(lambda review: review.reviewer_user_id, reviews))
```

Comment:
There is a much simpler way to do the same thing. Remeber that the best code is usually the simplest code.
eg:
```
return [o.reviewer_user_id for o in reviews]
```

Map returns an iterable instead of calculating all the values off the bat. Sometimes this is really useful if you are trying to be memory efficient. Or if the cvallback function takes long and you want to spread te workload out over a longer time perios. But in this case you are immediately taking that iterable and turining it into alist.

A list comprehention is simpler, more time and memory efficient (AFAIK) and easier to read. 

With your way we basically:
- create an object and a function
- iterate over the reviews and call the function on each review, function calls have overhead
- inside the function we use a getter to get the user id 

With a list comprehension we
- Dont declare any extra functinos or objects
- iterate over the reviews 
- get the user id with a perfectly normal getter 
- dont introduce function call overhead

Always remember the principle of KISS

Reply:
I originally had it as return [reviewer.reviewer_user_id for reviewer in reviewers] but I kept
   on getting an error that said that Django model object is not iterable, but
   now since I've destroyed my local database and changed it I no longer get that
   error, don't know why?  But I agree with you that the list/map business is
   not the best outcome.